include/zephyr/drivers/video.h

```diff
@@ -124,10 +124,14 @@ struct video_buffer {
  * Identify the video device endpoint.
  */
 enum video_endpoint_id {
-	VIDEO_EP_NONE,
-	VIDEO_EP_ANY,
-	VIDEO_EP_IN,
-	VIDEO_EP_OUT,
+	/** Targets some part of the video device not bound to an endpoint */
+	VIDEO_EP_NONE = -1,
+	/** Targets all input or output endpoints of the device */
+	VIDEO_EP_ALL = -2,
+	/** Targets all input endpoints of the device */
+	VIDEO_EP_IN = -3,
+	/** Targets all output endpoints of the device */
+	VIDEO_EP_OUT = -4,
```

Comment:
This does not encode any direction information into the `video_endpoint_id`, which allow using endpoint IDs exactly as in the manufacturer datasheet. Does it make sense?